hack/generate/samples/internal/go/memcached-with-webhooks/memcached_with_webhooks.go

```diff
@@ -754,7 +754,8 @@ const controllerImportsFragment = `
 `
 
 const controllerMonitoringImportFragment = `
-	"github.com/example/memcached-operator/monitoring"
+	// To enable operator custom metrics, uncomment all sections with 'monitoring'
```

Comment:
```suggestion
	// This sample has an example of how to create and work with custom metrics. The code is commented
	// in order to not break the basic tutorial to show authors how to build an operator. 
	// To enable operator custom metrics, uncomment all sections with 'monitoring'
```

Reply:
Could we try to give a little bit more context here @assafad, wdyt?

Reply:
Done